pkg/console/util_test.go

```diff
@@ -738,3 +1115,10 @@ func Test_identifyUniqueDisks(t *testing.T) {
 	assert.NoError(err, "expected no error while parsing disk data")
 	t.Log(out)
 }
+
+func Test_identifyUniqueDisksWithHarvesterInstall(t *testing.T) {
+	assert := require.New(t)
+	results, err := filterHarvesterInstallDisks([]byte(existingHarvesterInstalls))
+	assert.NoError(err, "expected no error while parsing disk data")
+	assert.Len(results, 1, "expected to find 1 disk from sample data")
```

Comment:
To confirm my understanding, in this case, the identified disk should be `sdd`, right?

Reply:
it will find `sdc` since both `sdc` and `sdd` are paths to the same device and sdc is found first when iterating through the list of disks